src/VirtualClient/VirtualClient.Actions.UnitTests/Examples/SysbenchOLTP/SysbenchOLTPExample.txt

```diff
@@ -0,0 +1,38 @@
+﻿sysbench 1.1.0-df89d34 (using bundled LuaJIT 2.1.0-beta3)
+
+Running the test with following options:
+Number of threads: 16
+Target transaction rate: 143/sec
+Initializing random number generator from current time
+
+
+Initializing worker threads...
+
+Threads started!
+
+SQL statistics:
+    queries performed:
+        read:                            5039772
+        write:                           259534
+        other:                           1284992
+        total:                           6584298
+    transactions:                        257521 (143.01 per sec.)
+    queries:                             6584298 (3657.94 per sec.)
+    ignored errors:                      0      (0.00 per sec.)
+    reconnects:                          0      (0.00 per sec.)
+
+Throughput:
+    events/s (eps):                      143.0086
+    time elapsed:                        1800.0319s
+    total number of events:              257521
+
+Latency (ms):
+         min:                                    7.39
+         avg:                                   28.97
+         max:                                  720.22
```

Comment:
Is there someway to tweak client so that we have more percentiles like P75 and P99

Reply:
no :/ i have no idea where the raw data comes from, and i don't think we can configure to see it...the only more granular data we could have is the 95p latency of every 10s interval